### DISCUSSION TITLE
I'm a bit confused here trying to setup custom increasing times....

### DISCUSSION THREAD

**Discussion by TiagoTiago** [fail2ban/fail2ban#3765] [Q&A]
I've tried way too many things on `jail.local`; but looking at the log the `incr` lines always say from the minimum time, it never builds up from a bigger amount seems to always go from the start (though the increased number does change sometimes, so maybe it's just not updating the first value just the "to"; but even then, still doesn't seem anyone is reaching high values); and looking at the jail status, no IP sticks around for long; but unless I did something wrong to do the counting, there's tons of repeated offenders in the log that should've accumulated a long ban time and still be listed as banned.

I've even tried increasing `findtime` to 48h and setting `maxretry` to just 1.

**Comment by sebres** [comment]
To be able to help you, one needs an example of the config you did (or an excerpt for `fail2ban-local -d` for the affected jail), excerpt from log filtered by an IP(s) you meant (few of the repeated offenders) and excerpt from fail2ban.log for the jail and that IPs.

Because currently you said absolutely **nothing**. Right now your story is totally senseless for someone.

Although... I hope your fail2ban version is at least 0.11 and you enabled `bantime.increment` for the jail or default section... As well as reloaded fail2ban after configuration change.

**Reply by TiagoTiago** [reply]
I'm sorry, you are right, I should've prepared better before posting; got a lot on my head right now, a bit overloaded. This is no emergency I think; looks so far the system by itself is holding on and this would just be an additional layer of protection and to make the system log looks cleaner;, for something that is just a hobby project at the moment. I'll see if I can get everything tidied up to come back with the logs and configurations and stuff in the next few days, without all the extra trash of the various trial and error attempts that could obscure what is causing the issue.
